pip/commands/install.py

```diff
@@ -251,12 +251,12 @@ def run(self, options, args):
                     progress_bar=options.progress_bar,
                 )
 
-                self.populate_requirement_set(
-                    requirement_set, args, options, finder, session, self.name,
-                    wheel_cache
-                )
-
                 try:
+                    self.populate_requirement_set(
```

Comment:
I think there was a test failure because of temporary files not getting cleaned up; it's in the `try` block so that the `finally` fires on failure and calls the cleanup. I don't remember the details, though, and I might have confused it with something else.